### PR TITLE
add note of tools package requirement for optimize window

### LINKED ISSUE #7897: Wrong docs or missing Optimize Window menu for warning
## Describe the bug

Wrong docs or missing menu:

image

image

## Expected behavior

Warning points user to existing menu.

## Your setup (please complete the following information)

- Unity Version 2018.4.22f1
- MRTK Version 2.4

### PR DESCRIPTION
**Pull request by davidkline-ms**
This change fixes #7897 by adding "Please note: The Mixed Reality Toolkit Tools package is required to use the Optimize Window." to the debug log messages related to the optimize window.

It also changes the warning about no spatializer to a simple message and fixes: #7949

### CODE REVIEW

**Review by Cameron-Micka — APPROVED** [review]
Looks great! Might be good to add a note in the docs too. (We recently did this with the AssetSwap tool too.)

File: Assets/MRTK/Core/Utilities/Editor/ScriptedImporterAssetReimporter.cs [hunk 1]
```diff
@@ -43,7 +43,7 @@ public static void OnPostprocessAllAssets(string[] importedAssets, string[] dele
 
                         if (numAttempts <= 3)
                         {
-                            Debug.LogWarning($"Asset '{asset}' appears to have failed importing, will attempt to re-import. Attempt: {numAttempts}");
+                            Debug.Log($"Asset '{asset}' appears to have failed importing, will attempt to re-import. Attempt: {numAttempts}");
```
Comment (keveleigh) [inline Assets/MRTK/Core/Utilities/Editor/ScriptedImporterAssetReimporter.cs:46]: I wonder if we can go a step further and _only_ log if the import fails `numAttempts` times? We don't really need to know that it's trying again...just that it failed

**Review by keveleigh — APPROVED** [review]

File: Assets/MRTK/Core/Utilities/Editor/ScriptedImporterAssetReimporter.cs (hunk 1 above)
Comment (davidkline-ms) [reply Assets/MRTK/Core/Utilities/Editor/ScriptedImporterAssetReimporter.cs]: good call. done.

File: Assets/MRTK/Core/Utilities/Editor/ScriptedImporterAssetReimporter.cs [hunk 2]
```diff
@@ -41,12 +41,7 @@ public static void OnPostprocessAllAssets(string[] importedAssets, string[] dele
 
                         assetsAttemptedToReimport[asset] = ++numAttempts;
 
-                        if (numAttempts <= 3)
-                        {
-                            Debug.LogWarning($"Asset '{asset}' appears to have failed importing, will attempt to re-import. Attempt: {numAttempts}");
-                            AssetDatabase.ImportAsset(asset);
```
Comment (keveleigh) [inline Assets/MRTK/Core/Utilities/Editor/ScriptedImporterAssetReimporter.cs:47]: Removing this line as well means that the re-import is never attempted

Comment (davidkline-ms) [reply Assets/MRTK/Core/Utilities/Editor/ScriptedImporterAssetReimporter.cs:47]: good catch!

Comment (keveleigh) [reply Assets/MRTK/Core/Utilities/Editor/ScriptedImporterAssetReimporter.cs:47]: I think just line 46 can be removed

**Review by keveleigh — CHANGES_REQUESTED** [review]

**Review by keveleigh — APPROVED** [review]

File: Documentation/Tools/OptimizeWindow.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Optimize window
 
-The MRTK Optimize Window is a utility to help automate and inform in the process of configuring a mixed reality project for best [performance](../Performance/PerfGettingStarted.md) in Unity. This tool generally focuses on rendering configurations that when set to the correct preset can save milliseconds of processing.
+The MRTK Optimize Window is a utility to help automate and inform in the process of configuring a mixed reality project for best [performance](../Performance/PerfGettingStarted.md) in Unity. This tool generally focuses on rendering configurations that when set to the correct preset can save milliseconds of processing. The utility is included with the `Microsoft.MixedReality.Toolkit.Unity.Tools` package.
```
Comment (keveleigh) [inline Documentation/Tools/OptimizeWindow.md:3]: Does our package have `.Unity` in the name?

Comment (keveleigh) [reply Documentation/Tools/OptimizeWindow.md:3]:
```suggestion
The MRTK Optimize Window is a utility to help automate and inform in the process of configuring a mixed reality project for best [performance](../Performance/PerfGettingStarted.md) in Unity. This tool generally focuses on rendering configurations that when set to the correct preset can save milliseconds of processing. The utility is included with the `Microsoft.MixedReality.Toolkit.Tools` package.
```

Comment (davidkline-ms) [reply Documentation/Tools/OptimizeWindow.md:3]: Yes. "Microsoft.MixedReality.Toolkit.Unity.Tools.2.5.3.unitypackage"

Comment (keveleigh) [reply Documentation/Tools/OptimizeWindow.md:3]: Ah! interesting. Should we clarify UPM vs .unitypackage, since they have different names?

**Pull request merged**